src/Component/RuleTable/RuleTable.tsx

```diff
@@ -115,7 +116,7 @@ export interface RuleTableProps extends Partial<RuleTableDefaultProps> {
   /** Properties of the SLD renderer */
   sldRendererProps?: SLDRendererAdditonalProps;
   /** Properties of the OpenLayers renderer */
-  oLRendererProps?: Partial<RendererProps>;
+  oLRendererProps?: Partial<OlRendererProps>;
```

Comment:
Do you think there is a way to just have a `rendererProps` property which is typed in dependency of the `renderType` property? Not blocking at all, but could be a nice feature.

Reply:
You mean something like `rendererProps<ol>` and `rendererProps<sld>`?